docs/cimpl6ToolingReference.md

```diff
@@ -342,83 +343,69 @@ The Content Profile (CP) specifies the profiles to be included in the IG. As exp
 
 ![Content profile scope](img_cimpl/Content-profile-scope.png)
 
-The CP provides several different types of information about the IG:
+The CP provides several types of information about the IG:
 
-* The list of classes (profiles) to be included in the IG,
-* The elements in these classes to be marked as [MustSupport](#Specifying-must-support-elements), designated with the `MS` tag, and
-* A list of classes _not_ to be profiled, designated with the [No-Profile](#specifying-no-profile-elements) `NP` tag.
+* The list of Entries (or entire namespaces) whose profiles are to be included in the IG,
+* A list of classes (or entire namespaces) _not_ to be profiled, and
+* Elements in the profiled classes to be marked as [MustSupport](#Specifying-must-support-elements).
```

Comment:
Profiles - if reference to FHIR profile s/b bold.  Consider/question for MustSupport
The list of Entries - CIMPL entry?  Code block.

Reply:
Hating the bolding of common words like profile more and more. But for now, I'll go with it. I feel like we are going to end up reverting to plain text for these words. The way we're doing it is ridiculous.